library/network/src/modules/SuSEFirewall.rb

```diff
@@ -42,6 +44,14 @@ def self.create
   end
 
   # ----------------------------------------------------------------------------
+  # SuSEFirewalld Class. Trying to provide relevent pieces of SF2 functionality via
```

Comment:
Would it be acceptable to split the classes into different files in the final commit instead of rewriting all 54 ones?